From ad786c7816ac12f9fe64b2f9684c2b45a4d3e579 Mon Sep 17 00:00:00 2001
From: Matthias Schiffer <mschiffer@universe-factory.net>
Date: Wed, 23 Feb 2022 23:41:08 +0100
Subject: [PATCH] modules: update routing packages

8f23999365de ahcpd: use SHA256 hash and use HTTPS everywhere
04e7f66aa770 mcproxy: Fix compilation with uClibc-ng
bf77e5f0e026 cjdns-tests: Don't build on ARC (#487)
01417ad6817a cjdns: Do not build on ARC
653ec59929cc olsrd: remove empty .gitignore file
3f095f7f7ef0 pimbd: add PKG_MIRROR_HASH
c1a2f7a25255 oonf-olsrd2: add PKG_MIRROR_HASH
2f9236d864ac oonf-dlep-radio: add PKG_MIRROR_HASH
20f3e5171029 oonf-dlep-proxy: add PKG_MIRROR_HASH
e05cb068c499 olsrd: add PKG_MIRROR_HASH
e37cf04da41a ohybridproxy: add PKG_MIRROR_HASH, refresh patch
95cd0ce4b5a1 mrd6: add PKG_MIRROR_HASH
00eacef47b5e minimalist-pcproxy: add PKG_MIRROR_HASH
84b7b3c55337 mcproxy: add PKG_MIRROR_HASH
f17815946a1b hnetd: add PKG_MIRROR_HASH, refresh patch
372ec6288639 cjdns: add PKG_MIRROR_HASH
ce5adaab9520 CI: fix multi-arch-test-build for 19.07 branch
2ed5a04ab819 CI: fix runtime testing for non master branch
c755ab8dce1c issuetemplate: reference foreign repos
cccff55a6288 CI: fix building multi-arch-test-build
63625f7e1987 CI: copy .github from openwrt-packages
df9f33c52277 batman-adv: Merge bugfixes from 2022.0
7a3e63fd45fe batman-adv: Merge bugfixes from 2021.4
c8734df67719 batman-adv: Refresh patches with quilt
3c10076826a5 batman-adv: Merge bugfixes from 2021.2
---
 modules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules b/modules
index 3885408a..211e5e76 100644
--- a/modules
+++ b/modules
@@ -10,7 +10,7 @@ PACKAGES_PACKAGES_COMMIT=5a842639dc87dab9daab34c064f6c8e268a34ba8
 
 PACKAGES_ROUTING_REPO=https://github.com/openwrt/routing.git
 PACKAGES_ROUTING_BRANCH=openwrt-19.07
-PACKAGES_ROUTING_COMMIT=101632e153b41238bc19dfd96ba2d23339dbcb76
+PACKAGES_ROUTING_COMMIT=8f23999365de1bf2617e03d3f536e4542258d985
 
 PACKAGES_GLUON_REPO=https://github.com/freifunk-gluon/packages.git
 PACKAGES_GLUON_BRANCH=v2021.1.x
-- 
2.17.1

